src/main/scala/nl/biopet/tools/fastqsplitter/FastqSplitter.scala

```diff
@@ -58,4 +79,33 @@ object FastqSplitter extends ToolCommand {
     reader.close()
     logger.info("Done, " + counter + " reads processed")
   }
+
+  def descriptionText: String =
+    """
+      |This tool divides a fastq file into smaller fastq files, based on the number of output files specified. For ecample,
+      |if one specifies 5 output files, it will split the fastq into 5 files of equal size. This can be very useful if one
+      |wants to use the chunking option in a pipeline: FastqSplitter can generate the exact number of fastq files
+      |(chunks) as needed.
+      |
+      |This tool will divide the fastq files in files of equal length.
```

Comment:
We should explain how the splitting is done, this is a unique method. Each file will have read from the beginning and the end of the original file. So there is no file position bias per chunk.